### PR TITLE
Fix for response code logging

### PR DESCRIPTION
**Pull request by BJap**
Changed logging so that all 200-range response codes log as info, not as error. (i.e. a response of 202 has been logging as an error)

### CODE REVIEW

**Comment by dimaj** [comment]
This does not cover all 2xx range codes: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success

**Comment by BJap** [comment]
Good point I didn't see those last time I added constants and I agree it's better as you said to add checks for them individually not as a range

**Review by trentontrees — COMMENTED** [review]
Rather than redefine all of the server code, we should move towards leveraging work of others: https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/HttpStatus.html

File: src/main/java/com/comcast/drivethru/api/HTTPRequestManager.java [hunk 1]
```diff
@@ -291,8 +308,8 @@ private ResponseContainer sendRequest(CloseableHttpClient client, Object request
             String responseLog = "Response: " + responseCode + " - ";
 
             if (responseText != null) responseLog += responseText;
-
-            if (responseCode == ServerStatusCodes.OK)
+            
+            if (SUCCESS_CODES.contains(responseCode))
```
Comment (trentontrees) [inline src/main/java/com/comcast/drivethru/api/HTTPRequestManager.java]: why not log all responses?

**Comment by BJap** [comment]
@trentontrees that's a good idea. I can do that possibly if desired?

**Comment by dimaj** [comment]
if you are referring to Trent's comment on offloading status codes to another library, you should probably hold off on that as it does not make much sense to import yet another heavy dependency just for status code definitions. Those status codes don't change very frequently, so I think we should be OK with the way things are right now.

**Comment by BJap** [comment]
@dimaj that dependency is already in the project. Looking at the code the only two that don't have a 1-1 mapping are 

`ALREADY_REPORTED 208`
and
`IM_USED 226`

**Pull request merged**